src/bridge/AdminRoomHandler.ts

```diff
@@ -158,11 +158,11 @@ export class AdminRoomHandler {
             case "!help":
                 await this.showHelp(adminRoom);
                 break;
-            default:
+            default: {
```

Comment:
What's up with this rule, the others don't need brackets?

Reply:
Oh this is for multi-line? Fair enough.

Reply:
It's for declaring variables inside a `case`, because case is not a block so it will be defined for the entire `switch`.
https://eslint.org/docs/rules/no-case-declarations